wordpress-mobile/WordPress-iOS
# Generalize Today Widget to Stats Widgets

This refactor will allow to use the existing extension of Today Widgets for all iOS 14 Stats Widgets.

The most important thing in this review (other than the testing that nothing is broken) is to look at the code as a whole to determine if the overall design needs improvements; the main goal being to host all three stats widgets in one extension.


Fixes #NA

To test:
The main goal of the testing is to make sure this did not break anything for the existing Today Widget.
- build/run and make sure you can install today widget and it works as expected.

PR submission checklist:
